nemo/collections/tts/models/hifigan.py

```diff
@@ -146,20 +154,20 @@ def training_step(self, batch, batch_idx, optimizer_idx):
             disc_real_outputs=msd_score_real, disc_generated_outputs=msd_score_gen
         )
         loss_d = loss_disc_msd + loss_disc_mpd
-        self.manual_backward(loss_d, self.optim_d)
+        self.manual_backward(loss_d)
```

Comment:
Why did you remove the optimizer? Does this still work?

Reply:
yes, this way was deprecated by PTL

Reply:
It looks like if we do this, then we have to manually called our optimizer.step(). I would advise you to add in the .step() call

Reply:
I call opt.step() in the next line

Reply:
Oh good point, I missed that.